source/slang/hlsl.meta.slang

```diff
@@ -6597,6 +6597,67 @@ matrix<T, N, M> sqrt(matrix<T, N, M> x)
     MATRIX_MAP_UNARY(T, N, M, sqrt, x);
 }
 
+// Cube root
+__generic<T : __BuiltinFloatingPointType>
+__target_intrinsic(cuda, "$P_cbrt($0)")
+[__readNone]
+T cbrt(T x)
+{
+    return pow(x, T(1) / T(3));
+}
+
+// Derivative of cube root
+__generic<T : __BuiltinFloatingPointType>
+[__readNone, Differentiable, ForwardDerivativeOf(cbrt)]
+DifferentialPair<T> cbrtFwdDiff(DifferentialPair<T> x)
```

Comment:
This should be implemented in diff.meta.slang, similar to how `sqrt` is implemented.